tests/test_wave_01.py

```diff
@@ -183,12 +188,12 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    #Assert Added
+    assert updated_data["watched"] == [FANTASY_2,movie_to_watch]
```

Comment:
```suggestion
    assert updated_data["watched"] == [FANTASY_2, movie_to_watch]
```
Missing white space

Reply:
Like my comment for the test above, I'd prefer more robust testing here. Since the function under test, `watch_movie`, does more than return a list, we should check that the `updated_data` on line 185 matches what we expect. 

The arrange section should create an `expected_result` dictionary that has one movie in the `watchlist` and two movies in the `watched` because `movie_to_watch` was watched and should be moved. 

```py
    # Arrange
    expected_result = {
        "watchlist": [FANTASY_1],
        "watched": [FANTASY_2, movie_to_watch]
    }

    # rest of your test

    # Assert
    assert updated_data == expected_result